liberty-maven-plugin/src/main/java/net/wasdev/wlp/maven/plugins/server/StopServerMojo.java

```diff
@@ -28,7 +28,7 @@
 public class StopServerMojo extends StartDebugMojoSupport {
 
     /**
-     * Timeout to verify stop successfully
+     * Timeout to verify stop successfully, this parameter is ignored
```

Comment:
@mattbsox Then I would recommend taking it out to avoid confusion in the future.  Make sure you test with a pom that has the setting in it, but you don't need to deliver that pom.